vic/drivers/shared_all/src/vic_time.c

```diff
@@ -30,9 +30,7 @@
  #include <float.h>
  #include <math.h>
```

Comment:
It looks like `float.h` and `math.h` are included in `vic_run.h`, which is included in `vic_driver_shared_all.h`